### ISSUE TITLE
Last Question no-answer="true", Form Does not submit

### ISSUE THREAD

**Issue by rahulkapoor86** [eduardotkoller/convForm#9]
Hi Edoardotkoller,

Thank you for this great plugin.

I am trying to implement it on my website, Where last question does not expect an answer (no-answer="true"). But in that case form does not submit.

For example at the last, once collected all the details, I would like to send a thanks message for which I dont expect an answer, but in this case form is not being submitted.

Please help I am stuck in between.

Thanks,
Rahul

**Comment by eduardotkoller** [comment]
Hello,

Sorry, this must be a bug. I think I didn't even think in submitting the form with a no-answer input because, technically, the user can't see the message if it did submit (it would print the message and instantly submit the form, so there is no time to read the message).

Will fix it as soon as I have time, thanks for the heads up!

**Comment by rahulkapoor86** [comment]
Hi Edoardotkoller,

Thanks for your reply.

I got your point. Is it not possible to submit the form and still keep all the messages there? May be submitting through Ajax. If we cant keep the messages there, then there is no point to send last text with "no-question" user cant read that (as you mentioned).

Actually I am trying to collect the name and email ID and then submitting it to Mailchimp and then at the last I want to tell user to check their email inbox for subscription/activation link, but user cant read the that last message if it clears the form immediately.

Let me know your opinion.
Look forward to your reply.

Thanks!
Rahul

**Comment by eduardotkoller** [comment]
If you want, until I correct the bug, you can just put the message on a simple input with no name and use a nonsense regex to not let the user submit.

**Comment by rahulkapoor86** [comment]
Hi Eduardotkoller,

Form submit via ajax ONLY if the **last question is  not a** no-answer="true". If the last question is no-answer="true" then form does not submit at all. The advantage of using ajax is that I can keep the form fields there.

Even if I use nonsense regex, in this case form wont be submitted at all.

I am going to use a temporary solution (till the time you fix the bug), i will use a hidden div and display it once the form is submitted.

Still wait for you to fix the bug, Hope you do this soon.

Thank you again for this plugin.

Rahul

**Comment by eduardotkoller** [comment]
Just fixed the bug and tested it, form is now submitting if last question is a `no-answer="true"`.

**Comment by rahulkapoor86** [comment]
Thanks for quick fix :)
